common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -349,6 +349,28 @@ static boolean equals(byte[] bytes1, int startPos1, int endPos1, byte[] bytes2,
         return bytes1[startPos1] == bytes2[startPos2];
     }
 
+    static int hashCode(byte[] bytes, int startPos, int endPos, long prime) {
```

Comment:
@Scottmitch can you add a test that shows that this function and `safeHashCode` are equivalent? (as good as this is possible with a test).